react/Breadcrumb.tsx

```diff
@@ -80,7 +80,7 @@ const Breadcrumb: React.FC<Props> = ({
   )
 
   return !navigationList.length ? null : (
-    <div className={`${styles.container} dn db-ns pv3`}>
```

Comment:
I don't think you need to leave the `db` class. If you just remove it, the `<div>` will be rendered at any screen size